jac-uk/admin
# #2051 User Able To Create Exercise With No Name

## What's included?
Use latest version of jac-kit which contains the fix for this bug.

This PR is linked to:
https://github.com/jac-uk/jac-kit/pull/126
https://github.com/jac-uk/jac-kit/pull/125

Closes #2051  

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

- Go to the Create Exercise page
- Click on on the Save and Continue button without filling in any fields and you will see two error messages (one for each field)
- Fill in the 'Do you want to add more information about this exercise now?' field
- Click on on the Save and Continue button and you will see a single error message for the name field
- Fill in both fields
- Click on on the Save and Continue button and the Details page will load with the newly created exercise

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
